operate/client/e2e-playwright/pages/Processes/FiltersPanel.ts

```diff
@@ -0,0 +1,187 @@
+import {Page, Locator, expect} from '@playwright/test';
+
+type OptionalFilter =
+  | 'Variable'
+  | 'Process Instance Key(s)'
+  | 'Parent Process Instance Key'
+  | 'Operation Id'
+  | 'Error Message'
+  | 'Start Date Range'
+  | 'End Date Range'
+  | 'Failed job but retries left';
+
+export class FiltersPanel {
+  private page: Page;
+  readonly panel: Locator;
+  readonly activeCheckbox: Locator;
+  readonly incidentsCheckbox: Locator;
+  readonly runningInstancesCheckbox: Locator;
+  readonly completedCheckbox: Locator;
+  readonly canceledCheckbox: Locator;
+  readonly finishedInstancesCheckbox: Locator;
+  readonly processNameFilter: Locator;
+  readonly processVersionFilter: Locator;
+  readonly processInstanceKeysFilter: Locator;
+  readonly parentProcessInstanceKey: Locator;
+  readonly flowNodeFilter: Locator;
+  readonly operationIdFilter: Locator;
+  readonly resetFiltersButton: Locator;
+  readonly errorMessageFilter: Locator;
+  readonly startDateFilter: Locator;
+  readonly variableNameFilter: Locator;
+  readonly variableValueFilter: Locator;
+  readonly deleteResourceButton: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.panel = page.getByRole('region', {name: 'Filter'});
+    this.activeCheckbox = this.panel.getByRole('checkbox', {name: 'Active'});
+    this.incidentsCheckbox = this.panel.getByRole('checkbox', {
+      name: 'Incidents',
+    });
+    this.runningInstancesCheckbox = this.panel.getByRole('checkbox', {
+      name: 'Running Instances',
+    });
+    this.completedCheckbox = this.panel.getByRole('checkbox', {
+      name: 'Completed',
+    });
+    this.canceledCheckbox = this.panel.getByRole('checkbox', {
+      name: 'Canceled',
+    });
+    this.finishedInstancesCheckbox = this.panel.getByRole('checkbox', {
+      name: 'Finished Instances',
+    });
+
+    this.processNameFilter = this.panel.getByRole('combobox', {
+      name: 'Name',
+    });
+
+    this.processVersionFilter = this.panel.getByRole('combobox', {
+      name: 'Version',
+    });
+
+    this.processInstanceKeysFilter = this.panel.getByLabel(
+      /^process instance key\(s\)$/i,
+    );
+
+    this.parentProcessInstanceKey = page.getByRole('textbox', {
+      name: /parent process instance key/i,
+    });
+
+    this.flowNodeFilter = this.panel.getByRole('combobox', {
+      name: /flow node/i,
+    });
+
+    this.operationIdFilter = this.panel.getByRole('textbox', {
+      name: /operation id/i,
+    });
+    this.resetFiltersButton = this.panel.getByRole('button', {
+      name: /reset filters/i,
+    });
+
+    this.errorMessageFilter = this.panel.getByRole('textbox', {
+      name: /error message/i,
+    });
+
+    this.startDateFilter = this.panel.getByRole('textbox', {
+      name: /start date range/i,
+    });
+
+    this.variableNameFilter = this.panel.getByRole('textbox', {
+      name: /name/i,
+    });
+
+    this.variableValueFilter = this.panel.getByRole('textbox', {
+      name: /value/i,
+    });
+
```

Comment:
Could we add a selector for the multiple mode toggle on the variable filter too?

Reply:
Good idea. I'll add it.